docs/source/content/supervised/logistic_regression.rst

```diff
@@ -9,48 +9,62 @@ Logistic regression is a method for binary classification.  It works to divide p
+Logistic regression works using a linear combination of inputs, so multiple information sources can govern the output of the model.  The parameters of the model are the weights of the various features, and represent their relative importance to the result.  In the equation that follows, you should recognize the formula used in linear regression.  Logistic regression is, at its base, a transformation from a linear predictor to a probability between 0 and 1.
 
-.. figure:: _img/Logistic_Eq.svg
+.. figure:: _img/WikiLogisticEQ.svg 
    :alt: Equation
 Ref: https://en.wikipedia.org/wiki/Logistic_regression
 
-The beta values are the weights and x values are the variable inputs.  This formula gives the probability that the input belongs to Class B, which is the output of the function.
+As in linear regression, the beta values are the weights and x values are the variable inputs.  This formula gives the probability that the input belongs to Class B, which is the goal of the logistic regression model.
+
+Motivation
+-----------------
+
+Why use logistic regression?  Logistic regression is well suited to the case of **binary classification**, or classifying in 2 categories.  
+Logistic regression is also a relatively simple method, utilizing a weighted sum of inputs, similar to linear regression.  
+Logistic regression is also useful in that it gives a continuous value, representing the probability of a given classification being correct.  
+For these reasons, advocates say that logistic regression should be the first_ thing learned in the data science world.
+
+.. _first: https://towardsdatascience.com/5-reasons-logistic-regression-should-be-the-first-thing-you-learn-when-become-a-data-scientist-fcaae46605c4
 
 Conclusion
 -----------------
 
-Machine learning libraries make using Logistic Regression very simple.  Check out the example code in the repository and follow along. The basic idea is to supply the training data as pairs of input and classification, and the model will be built automatically.  As always, keep in mind the basics mentioned in this repository, as there is no fool-proof method for machine learning.
+Logistic regression build upon linear regression by extending its use to classification.  
+Although it is not able to classify into more than two classes, it is still effective in what it does, and simple to implement.  
+Consider logistic regression as the first thought pass/fail method.  When you just need a pass/fail probability from data, logistic regression is the simplest and likely best option.
+
+Machine learning libraries make using Logistic Regression very simple.  Check out the example code in the repository and follow along. The basic idea is to supply the training data as pairs of input and classification, and the model will be built automatically.  As always, keep in mind the basics mentioned in the overview section of this repository, as there is no fool-proof method for machine learning.
```

Comment:
please add a section explaining the code implementation (also with a link to the code)